make/linux.mk

```diff
@@ -0,0 +1,34 @@
+LINUX_DIR = $(PLATFORM_DIR)/linux
+LINUX_PACKAGE:=koreader-linux-$(LINUX_ARCH_NAME)$(KODEDUG_SUFFIX)-$(VERSION).tar.xz
+
+update: all
+	mkdir -pv \
+		$(INSTALL_DIR)/linux/usr/bin \
+		$(INSTALL_DIR)/linux/usr/lib \
+		$(INSTALL_DIR)/linux/usr/share/pixmaps \
+		$(INSTALL_DIR)/linux/usr/share/applications \
+		$(INSTALL_DIR)/linux/usr/share/doc/koreader \
+		$(INSTALL_DIR)/linux/usr/share/man/man1
+	cp -pv resources/koreader.png $(INSTALL_DIR)/linux/usr/share/pixmaps
+	cp -pv $(LINUX_DIR)/koreader.desktop $(INSTALL_DIR)/linux/usr/share/applications
+	cp -pv $(LINUX_DIR)/copyright COPYING $(INSTALL_DIR)/linux/usr/share/doc/koreader
+	cp -pv $(LINUX_DIR)/koreader.sh $(INSTALL_DIR)/linux/usr/bin/koreader
+	cp -Lr $(INSTALL_DIR)/koreader $(INSTALL_DIR)/linux/usr/lib
+	gzip -cn9 $(LINUX_DIR)/koreader.1 > $(INSTALL_DIR)/linux/usr/share/man/man1/koreader.1.gz
+	chmod 644 \
+		$(INSTALL_DIR)/linux/usr/share/doc/koreader/copyright \
+		$(INSTALL_DIR)/linux/usr/share/man/man1/koreader.1.gz
+	rm -rf \
+		$(INSTALL_DIR)/linux/usr/lib/koreader/{ota,cache,clipboard,screenshots,spec,tools,resources/fonts,resources/icons/src}
+
+	# remove leftovers
+	find $(INSTALL_DIR)/linux -type f \( -name ".git" -o -name ".gitignore" -o -name "discovery2spore" -o -name "wadl2spore" -o -name "*.txt" -o -name "LICENSE*" -o -name "NOTICE" -o -name "README.md" \) -print0 | xargs -0 rm -rf
+	find $(INSTALL_DIR)/linux -type d \( -name "test" -o -name ".github" \) -print0 | xargs -0 rm -rf
+
+	# fix permissions
+	chmod -R u=rwX,og=rX $(INSTALL_DIR)/linux
+	XZ_OPT=9 tar -C $(INSTALL_DIR)/linux/usr -cvJf $(LINUX_PACKAGE) .
```

Comment:
Why use `$(INSTALL_DIR)/linux/usr/…` everywhere if we just end up stripping `/usr`?

Reply:
Good catch

> why?

For no good reason. :p